tensorflow/compiler/tf2xla/frontend_attributes_util.cc

```diff
@@ -24,15 +24,14 @@ const char kFrontendAttributesAttribute[] = "_XlaFrontendAttributes";
 }  // namespace
 
 xla::StatusOr<absl::optional<xla::FrontendAttributes>>
-GetFrontendAttributesFromNodeDef(const NodeDef& node_def) {
-  if (!HasNodeAttr(node_def, kFrontendAttributesAttribute)) {
-    return absl::optional<xla::FrontendAttributes>();
+GetFrontendAttributesFromNodeDef(const AttrSlice& attrs) {
+  auto attr = attrs.Find(kFrontendAttributesAttribute);
```

Comment:
Preferable to use concrete type here, the type isn't obvious from the RHS.

Reply:
Done